lib/puppet/provider/snmp_notification/cisco_ios.rb

```diff
@@ -44,10 +44,31 @@ def self.commands_from_instance(property_hash)
       commands_array
     end
 
-    def get(context, _names = nil)
+    def get(context, names = [])
```

Comment:
`names = nil` and `names = []` are two very different things in Resource API, please do not change the default.

See https://github.com/puppetlabs/puppet-specifications/blob/master/language/resource-api/README.md#provider-feature-simple_get_filter for the distinction.